CHANGES.rst

```diff
@@ -9,6 +9,11 @@
 
 - Add support for Python 3.8.
 
+- Drop support for legacy transaction APIs including
+  ``Transaction.register()`` and old ZODB3-style datamanagers. See
+  `issue 89
+  <https://github.com/zopefoundation/transaction/issues/89>`_.
```

Comment:
Should this result in an increased major version number?

Reply:
Yes. Luckily, it was [already bumped](https://github.com/zopefoundation/transaction/commit/d1e86d0e57e8d64ab477dc3191ff72d3f9094146) to 3 when dropping support for Python 3.4, and that hasn’t been released.